smart_fact_crawler/__init__.py

```diff
@@ -266,18 +256,19 @@ def main_page(url=None, timeout=None):
         humidity = Quantity(float('nan'), '%')
         wind_speed = Quantity(float('nan'), 'km/h')
 
-
+    run_id_in_system_status_pattern = r'.*\((\d*)\).*'
```

Comment:
Ah, right ... I had a `.*` there before and still had that in mind. 
Right so this is then a safe conversion. Cool